packages/core/src/translate.test.ts

```diff
@@ -0,0 +1,11 @@
+import { double, greet, initialize } from "@rose-lang/wasm";
+import { expect, test } from "vitest";
+
+initialize();
```

Comment:
Could you move this into `packages/wasm/index.js` so that it gets called automatically after we `await init`? Then we don't need to remember to call it here or in other usages.